app/views/Settings.js

```diff
@@ -68,7 +73,9 @@ class SettingsScreen extends Component {
   }
 
   eventHistoryButtonPressed() {
-    this.props.navigation.navigate('NotificationScreen');
+    if (this.state.isUserOffline) {
+      alert(languages.t('offlineMessage'));
```

Comment:
Add a test for this

Reply:
add a test means unit test case?

Reply:
Yes please, I will add base Settings.spec.js